pkgs/tools/networking/discord-sh/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenvNoCC, fetchFromGitHub, makeWrapper, curl, jq, coreutils }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "discord-sh";
+  version = "unstable-2022-05-19";
+
+  src = fetchFromGitHub {
+    owner = "ChaoticWeg";
+    repo = "discord.sh";
+    rev = "6aaea548f88eb48b7adeef824fbddac1c4749447";
+    sha256 = "sha256-RoPhn/Ot4ID1nEbZEz1bd2iq8g7mU2e7kwNRvZOD/pc=";
+  };
+
+  # The Makefile supplied by upstream wants to contact the internet, but
+  # mkDerivation wants to use it in the buildPhase by default. so we disable the
+  # buildPhase to ignore the makeFile.
+  dontBuild = "true";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  runtimeInputs = [
+    curl
+    jq
+    coreutils
+  ];
```

Comment:
This could be easily inlined. Also aren't coreutils part of standard PATH?

Reply:
Technically, but this is still an impurity that is dependent on the consumer's $PATH, which we are bypassing here. The way it is now, with `coreutils` specified means that even if the user has screwed with their $PATH, this script will continue to work. It's better to specify it like this. I also did it here github.com/NixOS/nixpkgs/pull/172373.

Reply:
I've inlined them.